28 April minutes (#187)
wintercg/admin: meetings/2026-04-28.md

* Approval of this meeting's agenda (2 min)
* Chair election [April 2026 chair election \#181](https://github.com/WinterTC55/admin/issues/181) (10 min)
* Runtime keys TR approval [Draft Technical Report runtime-keys\#30](https://github.com/WinterTC55/runtime-keys/pull/30) (20 min)
* Approval of additions to the runtime keys TR:
  * PythonMonkey [Add PythonMonkey runtime runtime-keys\#17](https://github.com/WinterTC55/runtime-keys/pull/17) (5 min)
  * QuickJS [Add QuickJS runtime-keys\#25](https://github.com/WinterTC55/runtime-keys/pull/25) (5 min)
  * QuickJS-NG [Add QuickJS-NG runtime-keys\#26](https://github.com/WinterTC55/runtime-keys/pull/26) (5 min)
  * Rhino [Add Rhino runtime-keys\#27](https://github.com/WinterTC55/runtime-keys/pull/27) (5 min)
  * Convex [Add Convex runtime-keys\#28](https://github.com/WinterTC55/runtime-keys/pull/28) (5 min)
  * Andromeda [Add Andromeda runtime-keys\#29](https://github.com/WinterTC55/runtime-keys/pull/29) (5 min)

# 4 Discussions

## 4.1 Runtime keys

### 4.1.1 Open PRs for additions to runtime keys

The following additions were addressed and approved by the committee:
* PythonMonkey
* QuickJS
* QuickJS-NG
* Rhino
* Convex

> 

Reviewer: ```suggestion
* Andromeda 

```

### 4.1.2 Document formatting

The secretary requested the committee consider an alteration to the rendering of the canonical source for runtime keys, without necessarily making changes to the underlying data, due to how the table of the current data renders.

This set off a conversation regarding how the data itself was organised, and the committee decided to ever-so-slightly modify the required information for submissions to the TR. Instead of ordering by organisation alphabetically, submissions to runtime-keys.json will be ordered by *key* alphabetically.

In addition, Ethan Arrowood volunteered to modify the document output to combine optional metadata into one table field for the purpose of rendering.

### 4.1.3 Remaining runtime additions

The committee addressed an open but outdated addition for Andromeda. Luckily, Dean Srebnik was on the call and able to provide updated information in real-time. Andromeda’s addition to the TR was approved.

## 4.2 Chair election

> Chris de Almeida volunteered to co-chair TC55 in place of Luca Casonato with his departure from Deno. Ethan Arrowood volunteered to assist in a facilitator role, and the committee will address again in the future when HarperDB joins Ecma.

Reviewer: ```suggestion
Chris de Almeida volunteered to co-chair TC55 in place of Luca Casonato with his departure from Deno. Ethan Arrowood volunteered to assist in a facilitator role, and the committee will address again in the future when Harper joins Ecma.
```

Reviewer: Company name is technically "HarperDB, Inc." and our DBA is "Harper" now.

  * Make sure you have reviewed this policy and, where applicable, so has your legal team
* Review of the notes for last meeting (5 min)
* Approval of this meeting's agenda (2 min)
* Chair election [April 2026 chair election \#181](https://github.com/WinterTC55/admin/issues/181) (10 min)
* Runtime keys TR approval [Draft Technical Report runtime-keys\#30](https://github.com/WinterTC55/runtime-keys/pull/30) (20 min)
* Approval of additions to the runtime keys TR:
  * PythonMonkey [Add PythonMonkey runtime runtime-keys\#17](https://github.com/WinterTC55/runtime-keys/pull/17) (5 min)
  * QuickJS [Add QuickJS runtime-keys\#25](https://github.com/WinterTC55/runtime-keys/pull/25) (5 min)
  * QuickJS-NG [Add QuickJS-NG runtime-keys\#26](https://github.com/WinterTC55/runtime-keys/pull/26) (5 min)
  * Rhino [Add Rhino runtime-keys\#27](https://github.com/WinterTC55/runtime-keys/pull/27) (5 min)
  * Convex [Add Convex runtime-keys\#28](https://github.com/WinterTC55/runtime-keys/pull/28) (5 min)
  * Andromeda [Add Andromeda runtime-keys\#29](https://github.com/WinterTC55/runtime-keys/pull/29) (5 min)

# 4 Discussions

> ## 4.1 Runtime keys

Reviewer 2: This section should mention that we approved the TR for the June GA.

This set off a conversation regarding how the data itself was organised, and the committee decided to ever-so-slightly modify the required information for submissions to the TR. Instead of ordering by organisation alphabetically, submissions to runtime-keys.json will be ordered by *key* alphabetically.

In addition, Ethan Arrowood volunteered to modify the document output to combine optional metadata into one table field for the purpose of rendering.

### 4.1.3 Remaining runtime additions

The committee addressed an open but outdated addition for Andromeda. Luckily, Dean Srebnik was on the call and able to provide updated information in real-time. Andromeda’s addition to the TR was approved.

## 4.2 Chair election

Chris de Almeida volunteered to co-chair TC55 in place of Luca Casonato with his departure from Deno. Ethan Arrowood volunteered to assist in a facilitator role, and the committee will address again in the future when Harper joins Ecma.

# 5 Any other business

> Andreu announced that he expects to take some extended time away starting mid-summer, the committee will concern itself with its leadership again at that time.

Reviewer 2: ```suggestion
Andreu announced that they expect to take some extended time away starting mid-summer, the committee will concern itself with its leadership again at that time.
```

### 4.1.1 Open PRs for additions to runtime keys

The following additions were addressed and approved by the committee:

* PythonMonkey
* QuickJS
* QuickJS-NG
* Rhino
* Convex
* Andromeda 

### 4.1.2 Document formatting

The secretary requested the committee consider an alteration to the rendering of the canonical source for runtime keys, without necessarily making changes to the underlying data, due to how the table of the current data renders.

This set off a conversation regarding how the data itself was organised, and the committee decided to ever-so-slightly modify the required information for submissions to the TR. Instead of ordering by organisation alphabetically, submissions to runtime-keys.json will be ordered by *key* alphabetically.

In addition, Ethan Arrowood volunteered to modify the document output to combine optional metadata into one table field for the purpose of rendering.

### 4.1.3 Remaining runtime additions

The committee addressed an open but outdated addition for Andromeda. Luckily, Dean Srebnik was on the call and able to provide updated information in real-time. Andromeda’s addition to the TR was approved.

### 4.1.4 Technical report approval

> The Runtime keys technical report was approved to be sent to the GA for adoption on 30 June.

Author: ```suggestion
The Runtime keys technical report was approved to be sent to the GA for adoption on 30 June. The committee is requesting that the [Ecma Alternative Copyright](https://ecma-international.org/policies/by-ipr/ecma-text-copyright-policy/#alternative-copyright-notice) be applied to the document.
```

Author: Apologies for forgetting this sentence in the initial submission.
